packages/examples/.storybook/preview.js

```diff
@@ -0,0 +1,39 @@
+// import fs from 'fs';
+// import path from 'path';
+import { initializeIcons } from '@uifabric/icons';
+import generateStoriesFromExamples from '@uifabric/build/storybook/generateStoriesFromExamples';
+import { configure, addParameters, addDecorator } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withA11y } from '@storybook/addon-a11y';
+
+addDecorator(withInfo());
+addDecorator(withA11y());
+addParameters({
+  a11y: {
+    manual: true
+  }
+});
+
+initializeIcons();
+
+// const packageName = path.basename(process.cwd());
```

Comment:
doesn't work since tgis runs on client and I got stuck on what to do instead

Reply:
I think the package name could either be a part of a path of the URL or a parameter. Then, you can use that information to FILTER out stories inside the loadStories() func.

Reply:
But I don't want to build all packages when only starting one package

Reply:
So I think the filtering should be done "server"/build side somehow